tests/testthat/testthat_mallows.R

```diff
@@ -1 +1,26 @@
-# add tests for mallows cp
+#library(testthat)
+#library(regscoreR)
+
+context("Testing mallows")
+
+# mallows
+test_that("mallows() returns Mallow's Cp of the model", {
+  # expected outputs:
+  expect_equal(round(mallow([1,2,3,4],[5,6,7,8],[1,2,3,5],3,2),5), 1.09375)
+})
+
```

Comment:
you wanna add cases with negative and floating numbers?

Reply:
you mean test the input is floating and negative numbers or not? or add cases with the negative output?

Reply:
I mean test the input with floating and negative numbers in y and y_pred.

Reply:
And maybe input with 0 in it. I don't think it matters much in our case, but maybe it does when we test with our function.